### PR TITLE
Update Cucumber in Gemfile.lock

### PR DESCRIPTION
**Pull request by juniortaeza**
### Desired Outcome

Resolves Snyk's medium severity security issue for potential HTML injection on outdated dependencies.

### Implemented Changes

Updates dependencies in Gemfile.lock, default Cucumber version in Gemfile, and runtime error causes from test scripts.

### Connected Issue/Story

CyberArk internal issue link: [CONJSE-1518](https://ca-il-jira.il.cyber-ark.com:8443/browse/CONJSE-1518)

### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [x] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [x] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [x] These changes are part of a larger initiative that will be reviewed later, or
- [ ] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [x] These changes are part of a larger initiative with a separate security review, or
- [ ] There are no security aspects to these changes

### CODE REVIEW

**Comment by szh** [comment]
@andytinkham Is there a way to verify this is fixed before merging?

**Comment by andytinkham** [comment]
> @andytinkham Is there a way to verify this is fixed before merging?

Not 100% conclusively, but running `snyk test` on the main branch lists multiple issues in cucumber and `snyk test` on update-branch does not, so I'm pretty confident it will.

**Comment by juniortaeza** [comment]
> > @andytinkham Is there a way to verify this is fixed before merging?
> 
> Not 100% conclusively, but running `snyk test` on the main branch lists multiple issues in cucumber and `snyk test` on update-branch does not, so I'm pretty confident it will.

glad to hear it!

File: features/support/world.rb [hunk 1]
```diff
@@ -6,6 +6,11 @@
 require 'uri'
 require 'securerandom'
 
+require_relative 'cf_helper'
+require_relative 'http_helper'
+require_relative 'conjur_helper'
```
[the remark this answers is not in the document]
Comment (juniortaeza) [reply features/support/world.rb:11]: sounds good! i'm confident this new loading scheme was necessary as RuntimeErrors would pop up without it – which is still weird as it seemingly looks like Ruby does do the auto-loading for you

**Review by john-odonnell — APPROVED** [review]
LGTM 👍

**Pull request merged**